packages/react/src/Text/Text.tsx

```diff
@@ -50,9 +50,16 @@ export type TextProps = {
    * Specify the text weight
    */
   weight?: TextWeightVariants | ResponsiveWeightMap
+  /**
+   * Specify the text alignment.
+   * Corresponds to the CSS `text-align` property.
+   * Note: Only applies to block elements.
+   */
+  align?: 'start' | 'center' | 'end' | 'left' | 'right'
```

Comment:
`start` and `end` are the same as `left` and `right` on left-to-right languages, should we only use logical properties to ensure consistency with other components such as SectionIntro and River, and implement good practices from the start to support right-to-left (RTL) languages in the future?

Reply:
@danielguillan - happy to remove those. Done in https://github.com/primer/brand/pull/341/commits/4c9665faedbb3636f4892931de7d25a53ca8c53d.

Mind taking another look please?